tests/core/v5_1/test_pool.py

```diff
@@ -9,6 +9,8 @@
 from ddht.v5_1.pool import Pool
 from ddht.v5_1.session import SessionInitiator, SessionRecipient
 
+TEST_CACHE_SESSION_SIZE = 128
```

Comment:
Might as well drop this even lower me thinks.  No need to perform a full 128 handshakes (384 packets over 256 sockets).  Maybe drop this to something like 16